### PR TITLE
Add NLCD Analysis colors to chart

### PR DESCRIPTION
**Pull request by mmcfarland**
Adds the NLCD code to the census output so that the chart will display the appropriate class color.

##### Test
* Navigate to a small area with several land types available
* Draw an area
* See that the reported land types are coded to their correct legend color (http://www.mrlc.gov/nlcd11_leg.php)

screenshot from 2015-09-22 11 13 15

Connects #786

### CODE REVIEW

File: src/mmw/apps/modeling/geoprocessing.py [hunk 1]
```diff
@@ -182,15 +184,21 @@ def update_rule(nlcd, soil, count, survey):
 
     def after_rule(count, survey):
         nlcd_names = [v[1] for v in NLCD_MAPPING.values()]
+        nlcd_keys = NLCD_MAPPING.keys()
```
Comment (mmcfarland) [inline src/mmw/apps/modeling/geoprocessing.py]: See [Python Documentation](https://docs.python.org/2/library/stdtypes.html#dict.items) which states that calles to `values` and `keys` (among others) will return in the same order, assuming no intervening modifications have been made.

**Comment by caseypt** [comment]
Got a failing test, but otherwise, +1.

screen shot 2015-09-22 at 11 50 09 am

screen shot 2015-09-22 at 11 50 33 am

**Comment by mmcfarland** [comment]
@azavea-bot rebuild

**Comment by mmcfarland** [comment]
@azavea-bot rebuild

**Pull request merged**